contracts/ContractRegistry.sol

```diff
@@ -0,0 +1,19 @@
+pragma solidity ^0.4.11;
+
+contract ContractRegistry {
```

Comment:
I'd add a comment specifying what this is used for, and the intended usage of the key => address mapping. For example, it's recommend that the key be the keccak hash of the name of the target contract.